lang/en/texts/discover.html

```diff
@@ -22,15 +22,18 @@ <h4 class="subheader">Be part of our collaborative, free and open database on fo
 		<img src="/images/svg/food-database-icon.svg" alt="A food products database" />
 		<h3>A food products database</h3>
 		<p>Open Food Facts is a database of food products with ingredients, allergens, nutrition facts and all the tidbits of information we can find on product labels.</p>
+		<p>	<a href="/open-food-facts-mobile-app?utm_source=off&utf_medium=web&utm_campaign=search_and_links_promo_en" class="button round " style="margin-bottom: 0">Get the app to start scanning & contributing</a>
+</p>
 	</div>
 	
 	<div class="medium-4 columns">
 		<img src="/images/svg/crowdsourcing-icon.svg" alt="Made by everyone - Crowdsourcing" />	
 		<h3>Made by everyone</h3>
 		<p>Open Food Facts is a non-profit association of volunteers.</p>
 		<p>100.000+ contributors like you have added 4&nbsp;000&nbsp;000+ products from 150 countries using our <a href="https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=en&utm_source=off&utm_campaign=discover-promo&utm_medium=website">Android</a> or <a href="https://apps.apple.com/app/open-food-facts/id588797948?utm_source=off&utm_campaign=discover-promo&utm_medium=website">iPhone</a> app or their camera to scan barcodes and upload pictures of products and their labels.
-	<a href="/open-food-facts-mobile-app?utm_source=off&utf_medium=web&utm_campaign=search_and_links_promo_en" class="button round " style="margin-bottom: 0">Get the app to start scanning & contributing</a>
 		</p>
+		<p>	<a href="https://wiki.openfoodfacts.org/Contributor?utm_source=off&utf_medium=web&utm_campaign=search_and_links_promo_en" class="button round " style="margin-bottom: 0">Get the app to start scanning & contributing</a>
```

Comment:
you're sending to the wiki ?

Reply:
Yes but I forgot to change the text ! I move the mobile app link to the other section.